var/spack/repos/builtin/packages/chill/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Chill(AutotoolsPackage):
```

Comment:
Why does this use Autotools?  Why not the CMake build for Chill?  Almost certainly, the Autotools build is / will be deprecated.  It would be better if Spack didn't go down that route and have to change later.

```
CHill can be built from two build systems, CMake or automake. (CMake is recomended for use with CLion)
```

Reply:
As of right now, the autotools build is the one that's been most thoroughly tested.